src/picotorrent/core/configuration.cpp

```diff
@@ -214,3 +216,30 @@ void Configuration::UpsertListenInterface(Configuration::ListenInterface const&
         stmt->Execute();
     }
 }
+
+bool Configuration::IsSystemDarkMode()
+{
+    bool systemUsesDarkTheme = false;
+    HKEY hKey = 0;
+    DWORD dwValue = 1;
+    DWORD dwBufSize = sizeof(dwValue);
+    if (RegOpenKeyEx(HKEY_CURRENT_USER, TEXT("Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize"), 0, KEY_QUERY_VALUE, &hKey) == ERROR_SUCCESS)
```

Comment:
This call must be matched with a call to `RegCloseKey` as well to avoid leaking HKEY handles. Probably makes sense to close it right after we've queried the value.